app/migrations/20250306122357-adds_organization_table.cjs

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable("Organization", {
+      organization_id: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        primaryKey: true,
+      },
+      name: {
+        type: Sequelize.TEXT,
+        allowNull: false,
+      },
+      contact_email: {
+        type: Sequelize.TEXT,
+        allowNull: false,
+      },
+      contact_number: {
+        type: Sequelize.TEXT,
+        allowNull: true,
+      },
+      user_id: {
+        type: Sequelize.UUID,
+        allowNull: true,
+        foreignKey: true,
+        references: {
+          model: "User",
+          key: "user_id",
+        },
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
```

Comment:
Let's make sure we're following the conventions of the other models and use `snake_case` for the attribute names in the DB (which means you need to define that these columns are used in the Sequelize model, see the other models for a reference).